google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/CountErrorsPerInterceptorTask.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.data.v2.stub;
+
+import com.google.cloud.bigtable.stats.StatsRecorderWrapper;
+import com.google.cloud.bigtable.stats.StatsWrapper;
+import com.google.common.collect.ImmutableMap;
+import java.util.Set;
+
+class CountErrorsPerInterceptorTask implements Runnable {
+  private final Set<ConnectionErrorCountInterceptor> interceptors;
+  private StatsRecorderWrapper statsRecorderWrapper;
+
+  public void setStatsRecorderWrapper(StatsRecorderWrapper statsRecorderWrapper) {
+    this.statsRecorderWrapper = statsRecorderWrapper;
+  }
+
+  public CountErrorsPerInterceptorTask(
+      Set<ConnectionErrorCountInterceptor> interceptors,
+      ImmutableMap<String, String> builtinAttributes) {
+    this.interceptors = interceptors;
+    this.statsRecorderWrapper = StatsWrapper.createRecorder(null, null, builtinAttributes);
```

Comment:
Also add a comment here, something like: "We're only interacting with putAndRecordPerConnectionErrorCount method so OperationType and SpanName won't matter"